src/librustdoc/passes/strip_hidden.rs

```diff
@@ -89,6 +88,26 @@ impl Stripper<'_, '_> {
 impl DocFolder for Stripper<'_, '_> {
     fn fold_item(&mut self, i: Item) -> Option<Item> {
         let has_doc_hidden = i.is_doc_hidden();
+
+        if let clean::ImportItem(clean::Import { source, .. }) = &i.kind {
+            if let Some(source_did) = source.did
```

Comment:
```suggestion
        if let clean::ImportItem(clean::Import { source, .. }) = &i.kind
            && let Some(source_did) = source.did
```

Let's remove one indentation level too.